dockers/docker-database/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# first, create all redis configuration files
+create_all_redis_config
+
+# then, generate all redis server supervisord configuration file
+sonic-cfggen -j /etc/sonic/database_config.json -t /usr/share/sonic/templates/supervisord_database.conf.j2 > /etc/supervisor/conf.d/supervisord_database.conf
+
+# finally, reread supervisord config files and start new added redis servers only
+supervisorctl reread
+supervisorctl update
```

Comment:
Static for each image, and we are not allowed to change it after deployed. We can render templates during building image.

Reply:
We don't want static config for each image, this is the conclusion after the discussion in the community meeting, Guohan prefer to have a way to modify the database_config.json file and reload to make it take affect. You can double check with Guohan and make sure we are on the same understanding.

Reply:
can you follow this pattern ?

https://github.com/Azure/sonic-buildimage/commit/a6edef2fa52f5b7771cc5751cde1a190d96ffa50#diff-21ff3534aafbd7df004ceb3e24a6b12e

you no longer need to reread and update.

Reply:
if we changed the ENTRY POINT to a shell like this, then we don't need to reread and update. 
Actually I did this way when I first send the PR , and Qi reviewed and said we should keep supervisord as ENTRY_POINT, hence modified as what we have today.  I 'll change it back and let you know.